syntax/listblock.php

```diff
@@ -57,23 +59,22 @@ function handle($match, $state, $pos, Doku_Handler $handler) {
         switch ($state) {
             case DOKU_LEXER_ENTER:
                 $this->eventhandler->notifyEvent('open', 'list', NULL, $pos, $match, $handler);
-                $ReWriter = new Doku_Handler_List($handler->CallWriter);
-                $ReWriter = new Doku_Handler_Creole_List($handler->CallWriter);
-                $handler->CallWriter = & $ReWriter;
-                $handler->_addCall('list_open', array($match), $pos);
+                $ReWriter = new Doku_Handler_Creole_List($handler->getCallWriter());
+                $handler->setCallWriter($ReWriter);
+                $handler->addCall('list_open', array($match), $pos);
                 break;
             case DOKU_LEXER_EXIT:
                 $this->eventhandler->notifyEvent('close', 'list', NULL, $pos, $match, $handler);
-                $handler->_addCall('list_close', array(), $pos);
-                $handler->CallWriter->process();
-                $ReWriter = & $handler->CallWriter;
-                $handler->CallWriter = & $ReWriter->CallWriter;
+                $handler->addCall('list_close', array(), $pos);
+                $ReWriter = & $handler->getCallWriter();
```

Comment:
This function returns a object, in PHP these are default by reference. So the & is not needed anymore.

Reply:
thanks, fixed